tests/unit/nupic/algorithms/anomaly_likelihood_test.py

```diff
@@ -231,10 +231,10 @@ def testEstimateAnomalyLikelihoodsMalformedRecords(self):
 
     # Generate an estimate using fake distribution of anomaly scores.
     data1 = _generateSampleData(mean=0.2)
-    data1 = data1 + [(2, 2), (2, 2, 2, 2), (), (2)]  # Malformed records
+    data1 = data1[0:1000] + [(2, 2), (2, 2, 2, 2), (), (2)]  # Malformed records
 
     likelihoods, avgRecordList, estimatorParams = (
-      an.estimateAnomalyLikelihoods(data1[0:1000])
+      an.estimateAnomalyLikelihoods(data1[0:1004])
     )
     self.assertEqual(len(likelihoods), 1000)
```

Comment:
here it's 1004 https://github.com/numenta/nupic/pull/1855/files#diff-938a3b6b5c718f261186df565988e7a4R237 
while there is 1000 https://github.com/numenta/nupic/pull/1855/files#diff-938a3b6b5c718f261186df565988e7a4R240 

or do I miss something?